output.c

```diff
@@ -42,9 +42,9 @@
 #include "xwayland.h"
 #endif
 
-#define OUTPUT_CONFIG_UPDATED                                                                                          \
-	(WLR_OUTPUT_STATE_ENABLED | WLR_OUTPUT_STATE_SCALE | WLR_OUTPUT_STATE_TRANSFORM |                              \
-	 WLR_OUTPUT_STATE_ADAPTIVE_SYNC_ENABLED)
+#define OUTPUT_CONFIG_UPDATED (WLR_OUTPUT_STATE_ENABLED | \
+	WLR_OUTPUT_STATE_MODE | WLR_OUTPUT_STATE_SCALE | \
```

Comment:
Is it not already managed by `handle_output_mode()`?

Reply:
Yeah, it is, and probably doesn't make sense to land outside of https://github.com/cage-kiosk/cage/pull/281...